lib/connect.js

```diff
@@ -666,6 +663,26 @@ regular).
         return  _get("order.trades", {"order_id": order_id}, null, formatResponse);
     };
 
+    /**
+     * Fetch required margin for order/list of orders
+     * @method orderMargins
+     * @memberOf KiteConnect
+     * @instance
+     * @param {Object} params   Margin fetch order params.
+     * @param {string} params.exchange    Name of the exchange(eg. NSE, BSE, NFO, CDS, MCX)
+     * @param {string} params.tradingsymbol   Trading symbol of the instrument
+     * @param {string} params.transaction_type   eg. BUY, SELL  
+     * @param {string} params.variety   Order variety (regular, amo, bo, co etc.)
+     * @param {string} params.product   Margin product to use for the order
+     * @param {string} params.order_type   Order type (MARKET, LIMIT etc.)
+     * @param {number} params.quantity   Quantity of the order
+     * @param {number} params.price   Price at which the order is going to be placed (LIMIT orders)
+     * @param {number} params.trigger_price   Trigger price (for SL, SL-M, CO orders)
+     */
+    self.orderMargins = function(params){
+        return _post("order.margins", params, ...Array(2), true);
```

Comment:
I didn't get what `...Array(2)` does here? may be I am missing something.

Reply:
...Array(2) sends blank/null value to **responseType** and **responseTransformer** parameter of **_post** function. In order to send true flag for optional json parameter.

Reply:
you can send `null` as well I guess.